frontend/test/cypress/integration/product-list/parts_page_search.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('parts page search', () => {
+   const user = cy;
+
+   beforeEach(() => {
+      user.intercept('/1/indexes/**').as('search');
+      user.visit('/Parts');
+   });
+
+   it('should search for products', () => {
```

Comment:
I prefer to split tests like this into two because they are two different user flows. Thoughts?

```
should show results when the search term exists
should show no results when search term doesn't exist
```

Reply:
Yes, I agree!